### PR TITLE
N3 patch

### PR DESCRIPTION
**Pull request by michielbdejong**
Adapt to Solid spec 0.9 which uses a slightly different PATCH syntax.

### CODE REVIEW

**Review by bourgeoa — APPROVED** [review]
LGTM tested on https://solid-crud-tests-example-2.solidcommunity.net

File: test/surface/update.test.ts [hunk 1]
```diff
@@ -437,14 +442,16 @@ describe('Update', () => {
         }
       });
       const headers = {
-        'Content-Type': 'application/sparql-update',
+        'Content-Type': 'application/text/n3',
```
Comment (csarven) [inline test/surface/update.test.ts:445]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 2]
```diff
@@ -473,14 +480,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:483]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 3]
```diff
@@ -509,14 +518,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:521]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 4]
```diff
@@ -545,14 +556,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:559]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 5]
```diff
@@ -585,9 +598,11 @@ describe('Update', () => {
       });
       const result = await solidLogicBob.fetch(resourceUrl, {
         method: 'PATCH',
-        body: 'DELETE DATA { <#hello> <#linked> <#world> . }',
+        body: "@prefix solid: <http://www.w3.org/ns/solid/terms#>." +
+        "#patch a solid:InsertDeletePatch;" +
+        "  solid:deletes { <#hello> <#linked> <#world> . }",
         headers: {
-          'Content-Type': 'application/sparql-update'
+          'Content-Type': 'application/text/n3'
```
Comment (csarven) [inline test/surface/update.test.ts:605]:
```suggestion
          'Content-Type': 'text/n3'
```

File: test/surface/update.test.ts [hunk 6]
```diff
@@ -615,14 +630,16 @@ describe('Update', () => {
         }
       });
       const headers = {
-        'Content-Type': 'application/sparql-update',
+        'Content-Type': 'application/text/n3',
```
Comment (csarven) [inline test/surface/update.test.ts:633]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 7]
```diff
@@ -651,14 +668,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:671]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 8]
```diff
@@ -687,14 +706,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:709]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 9]
```diff
@@ -723,14 +744,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:747]:
```suggestion
        'Content-Type': 'text/n3',
```

File: test/surface/update.test.ts [hunk 10]
```diff
@@ -761,14 +784,16 @@ describe('Update', () => {
...
```
Comment (csarven) [inline test/surface/update.test.ts:787]:
```suggestion
        'Content-Type': 'text/n3',
```

**Pull request merged**